cairo/src/enums.rs

```diff
@@ -42,6 +42,13 @@ macro_rules! gvalue_impl {
                 <Self as glib::StaticType>::static_type()
             }
         }
+
+        impl From<$name> for glib::Value {
```

Comment:
This also needs a gir change to implement the same there for enums/flags or not?

Reply:
Yes: https://github.com/gtk-rs/gir/pull/1402

Reply:
Can you update this PR with that?

Reply:
Updated, but something else is breaking in `gio::Icon`: https://github.com/gtk-rs/gtk-rs-core/commit/245a088c38166343a88807e8b564a542724138b7#diff-56fdede3ae1018d2c5e30d97bd2f117b17c659689e9bee7818680e735625d932

```
error[E0283]: type annotations needed
  --> gio/src/auto/icon.rs:31:65
   |
31 |                 ToGlibPtr::<*mut ffi::GIcon>::to_glib_none(self.as_ref()).0
```

Reply:
That should be fixed by https://github.com/gtk-rs/gtk-rs-core/pull/829#discussion_r1027121550

Reply:
Hmm, but without the gir-files update it stays in the `impl Icon` instead of going into the trait impl so it shouldn't have gotten the `as_ref()` added.

Reply:
I'll take a look at this after you solved the other discussion item here. I'll get that all fixed and merged then :)

Reply:
Ok, so my job now to get these two PRs in :) Thanks!

Reply:
I think this was my bad, I updated the gir-files and now it's working. This looks merge ready now